spec.html

```diff
@@ -11824,7 +11824,7 @@
           TemplateMiddleList[?Yield, ?Await, ?Tagged] TemplateMiddle Expression[+In, ?Yield, ?Await]
       </emu-grammar>
 
-      <emu-clause id="sec-static-semantics-template-early-errors">
+      <emu-clause id="sec-static-semantics-template-early-errors" oldids="sec-primary-expression-template-literals-static-semantics-early-errors">
```

Comment:
Is old-ids a thing I forgot? I think the convention here is <a id=oldid></a> in the proper place.